README.md

```diff
@@ -36,7 +32,7 @@ For more information please visit the [API documentation website](https://acinq.
 
 ## Documentation
 
-Please visit our [wiki](https://github.com/acinq/eclair/wiki) to find detailed instructions on how to configure your
+Please visit our [docs](https://github.com/lightningnetwork/lightning-rfc/issues/769) and [wiki](https://github.com/acinq/eclair/wiki) to find detailed instructions on how to configure your
```

Comment:
Copy paste error?

Reply:
Wow oh yeah that went very wrong

Reply:
Is there a reason why you are not using a relative link?

Reply:
I think you can't have a relative link to a folder, only to a file, can you?

Reply:
But I can add an index/readme file inside the `docs` folder and point to that if you prefer

Reply:
Apparently you can (see https://github.com/ACINQ/eclair/tree/readme-relative-folder)

Reply:
Nice!